src/init.c

```diff
@@ -213,27 +212,21 @@ shmem_internal_init(int tl_requested, int *tl_provided)
     /* create symmetric heap */
     ret = shmem_internal_symmetric_init(heap_size, heap_use_malloc);
     if (0 != ret) {
-        fprintf(stderr,
-                "[%03d] ERROR: symmetric heap initialization failed: %d\n",
-                shmem_internal_my_pe, ret);
+        RETURN_ERROR_MSG("symmetric heap initialization failed: %d\n", ret);
```

Comment:
There isn't really an explanation; it's just for debugging so we can trace back the cause of the error.